v19.1/recommended-production-settings.md

```diff
@@ -43,25 +43,35 @@ For added context about CockroachDB's fault tolerance and automated repair capab
 
 ## Hardware
 
+### Terminology
+
+To properly select your cluster's hardware, it's important to review define basic hardware terminology:
+
+Term | Definition
+-----|------------
+**CPU** | The [physical central processing unit (CPU)](https://en.wikipedia.org/wiki/Central_processing_unit) that runs a CockroachDB server run on a (virtual) machine.</br>Also known as: central processor, main processor
+**vCPU** | Four virtual central processing units (vCPU) comprise a CPU. vCPUs increase a core's performance by running multiple tasks simultaneously. </br>Also known as: hyperthread, scheduling unit
+**Core** | A core is the basic unit for processing instructions in a CPU. A rule of thumb for cloud providers is that there are typically 2 vCPUs per core.
```

Comment:
Not just cloud providers. Most server-class processors these days have two vCPUs per core.